go/config/BUILD.bazel

```diff
@@ -79,3 +79,14 @@ string_list_flag(
     build_setting_default = [],
     visibility = ["//visibility:public"],
 )
+
+label_flag(
+    name = "pgoprofile",
+    build_setting_default = ":empty",
+    visibility = ["//visibility:public"],
+)
+
+filegroup(
+    name = "empty",
+    visibility = ["//visibility:public"],
```

Comment:
Can this be made private?

Reply:
No, it can't be private since it is the default value for go_binary. 

```
ERROR: /home/preston/.cache/bazel/_bazel_preston/996ab6b61f6a3f8e34862586db434593/external/org_golang_x_tools/cmd/fiximports/BUILD.bazel:11:10: in go_binary rule @org_golang_x_tools//cmd/fiximports:fiximports: target '//go/config:empty' is not visible from target '@org_golang_x_tools//cmd/fiximports:fiximports'. Check the visibility declaration of the former target if you think the dependency is legitimate
```